### ISSUE TITLE
Cannot read Notbook Section or Page titles in OneNote

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#6601]
In OneNote 2016 at least, you can press:

CONTROL+TAB / CONTROL+SHIFT+TAB to move between sections in the current notebook (visually they appear similar to "tabs" above the work area and below the ribbon).
CONTROL+PAGE UP / CONTROL+PAGE DOWN to move between pages in the current section of the notebook (visually they appear in a list at the right of screen).

The focus moves accordingly, however, NVDA does not announce the new location or that the focus has moved (unless it moves directly into a text area in the new page or section in which case it reads the current line).

NVDA+T / NVDA+TAB does not give any information about the current page or section.

**Comment by feerrenrut** [comment]
I can reproduce this in OneNote 2013 also.

**Comment by Qchristensen** [comment]
Note that was using the desktop version of OneNote.  The Modern App version of OneNote 2016 puts pages AND sections all together in a list on the left hand side of the current page working area and control+page up / control+page down move between current pages in the section before the other sections.  This issue is actually consistent across both versions (as in, I can't read current page / section either with NVDA+TAB / NVDA+T or when moving with control+page up / control+page down).

**Comment by PratikP1** [comment]
It would be good to have full support for OneNote. Microsoft is doing some impressive things with it.

**Comment by Adriani90** [comment]
Yes, I have also created some issues regarding NVDA and onenote. Adding support for it would be reasonable for many users.

**Comment by Adriani90** [comment]
Updates with NVDA 2024.4 RC and ONenote for MS Office 365 build 2409 on Windows 11 23 H2:
Pressing nvda+t reports now the book title, and nvda+tab reports the type of content element (e.g. cell data for a table cell).
However, NVDA still doesn't announce the page titles when moving with ctrl+ or ctrl+shift+tab or ctrl+pg up and down.
